community/ocaml/APKBUILD

```diff
@@ -102,8 +96,6 @@ runtime() {
 		'*.cma' \
 		'VERSION' \
 		'stublibs' \
-		'vmthreads/*.cmi' \
-		'vmthreads/*.cma' \
```

Comment:
The vmthreads library is no longer available. This library was deprecated in 4.08.0.